zrml/prediction-markets/Cargo.toml

```diff
@@ -13,14 +13,14 @@ zrml-court = { default-features = false, path = "../court" }
 zrml-liquidity-mining = { default-features = false, path = "../liquidity-mining" }
 zrml-market-commons = { default-features = false, path = "../market-commons" }
 zrml-simple-disputes = { default-features = false, path = "../simple-disputes" }
+pallet-timestamp = { branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com/purestake/substrate", default-features = false }
```

Comment:
if you are referencing this for benchmarks then we need to add mocks feature to runtime-benchmarks feature so that mocks gets build with benchmarks too. I tried that but it felt bit complicated so I choose this path.